share/completions/brew.fish

```diff
@@ -207,6 +234,20 @@ complete -f -c brew -n '__fish_brew_using_command search' -l fink -d 'Search on
 complete -f -c brew -n '__fish_brew_using_command -S' -l macports -d 'Search on MacPorts'
 complete -f -c brew -n '__fish_brew_using_command -S' -l fink -d 'Search on Fink'
 
+# services
+complete -f -c brew -n '__fish_brew_needs_command' -a services -d 'Manage Homebrew services'
+complete -f -c brew -n '__fish_brew_needs_services_action' -a cleanup -d 'Get rid of stale services and unused plist'
+complete -f -c brew -n '__fish_brew_needs_services_action' -a list -d 'List all services managed by Homebrew'
+complete -f -c brew -n '__fish_brew_needs_services_action' -a restart -d 'Gracefully restart a service'
+complete -f -c brew -n '__fish_brew_needs_services_action' -a start -d 'Start a service'
+complete -f -c brew -n '__fish_brew_needs_services_action' -a stop -d 'Stop a service'
+complete -f -c brew -n '__fish_brew_is_subcommand_services restart' -a '(__fish_brew_services)' -d 'formula'
+complete -f -c brew -n '__fish_brew_is_subcommand_services start' -a '(__fish_brew_services)' -d 'formula'
+complete -f -c brew -n '__fish_brew_is_subcommand_services stop' -a '(__fish_brew_services)' -d 'formula'
```

Comment:
Since these all take the same arguments, it might be worth it to make `__fish_brew_is_subcommand_services` accept a list and do

```fish
complete -f -c brew -n '__fish_brew_is_subcommand_services restart start stop'
```